lib/tapioca/loader.rb

```diff
@@ -29,6 +29,15 @@ def load_bundle(initialize_file, require_file)
     sig { returns(Tapioca::Gemfile) }
     attr_reader :gemfile
 
+    sig { params(file: T.nilable(String)).void }
```

Comment:
Nilable for client convenience?

Reply:
Yeah. mostly. This method checks if the passed argument is `nil` and short-circuits so that the call-site can just ignore that detail.